include/mapbox/geojson/rapidjson.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <rapidjson/document.h>
+#include <mapbox/geometry.hpp>
+#include <mapbox/variant.hpp>
+
+namespace mapbox {
+namespace geojson {
+
+using value               = mapbox::geometry::value;
+using point               = mapbox::geometry::point<double>;
+using geometry            = mapbox::geometry::geometry<double>;
+using feature             = mapbox::geometry::feature<double>;
+using feature_collection  = mapbox::geometry::feature_collection<double>;
+using json_value          = rapidjson::GenericValue<rapidjson::UTF8<>, rapidjson::CrtAllocator>;
+using prop_map            = std::unordered_map<std::string, value>;
+using geojson             = mapbox::util::variant<geometry, feature, feature_collection>;
+
+geojson convert(const json_value &json);
```

Comment:
I think the declarations here can look similar to `geojson.hpp`:

```C++
// Convert inputs of known types. Instantiations are provided for geometry, feature, and
// feature_collection.
template <class T>
T convert(const rapidjson_value &);

// Convert any GeoJSON type.
geojson parse(const rapidjson_value &);
```

Unless for some reason they are needed by mapbox-gl-native, let's hold on exposing conversions for `value`, `prop_map`, and individual geometry types for now.